rmf_robot_sim_common/include/rmf_robot_sim_common/utils.hpp

```diff
@@ -60,7 +60,8 @@ double compute_ds(
   const double v_max,
   const double accel_nom,
   const double accel_max,
-  const double dt);
+  const double dt,
+  const double v_target = 0.0);
```

Comment:
Nit, there was some discussion here about the `compute_ds` and `compute_desired_rate_of_change` being basically the same function, can we take the chance to remove one of the two?

Reply:
There's a line that differs between functions by double the divisor at https://github.com/open-rmf/rmf_simulation/pull/41/files/d55cfacfe9310deef0b6012e1a81ab2c7bb51394#diff-3bd4615902d61050ebeb9f584b212fc5d5ef07c1eed12cb5fa7814dbed7d59a4R41
but you're right let me remove it and test it with the other robots

Reply:
delayed, will have to tweak the turning_right_angle_mul_offset values again